apis/route_handler_test.go

```diff
@@ -131,40 +131,42 @@ func testRouteHandler(t *testing.T, when spec.G, it spec.S) {
 		})
 
 		it("returns the Route in the response", func() {
-			expectedBody, err := json.Marshal(presenters.RouteResponse{
-				GUID:     "test-route-guid",
-				Protocol: "http",
-				Host:     "test-route-name",
-				URL:      "test-route-name.example.org",
-				Relationships: presenters.Relationships{
-					"space": presenters.Relationship{
-						GUID: "test-space-guid",
-					},
-					"domain": presenters.Relationship{
-						GUID: "test-domain-guid",
+			expectedBody := `{
```

Comment:
We should decide upon a pattern for this sort of thing. While a raw string can be a better bellwether because it is inherently brittle, that brittleness can be a pain, as well. Either way, consistency should be key. Future topic to discuss.